packages/components/src/__future__/Tag/Tag.tsx

```diff
@@ -1,58 +1,37 @@
 import React, { ReactNode, HTMLAttributes } from "react"
 import classNames from "classnames"
 import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./Tag.module.scss"
 
-export type TagColors =
-  | "gray"
-  | "blue"
-  | "green"
-  | "yellow"
-  | "orange"
-  | "red"
-  | "purple"
+export const TagColorKeys = [
+  "gray",
+  "blue",
+  "green",
+  "yellow",
+  "orange",
+  "red",
+  "purple",
+] as const
+
+export type TagColors = (typeof TagColorKeys)[number]
```

Comment:
To prevent confusion for our consumers, we don't want to export `TagColorKeys` from the entrypoint.
Create a `Tag/types.ts`, and move `TagColorKeys` and `TagColors` there, then in `Tag/index.ts` export only `TagColors` from the new file.